api-docs/.eleventy.js

```diff
@@ -0,0 +1,75 @@
+import syntaxHighlightPlugin from '@11ty/eleventy-plugin-syntaxhighlight';
+import markdownIt from 'markdown-it';
+import markdownItAnchor from 'markdown-it-anchor';
+import { parse } from 'node-html-parser';
+
+export default function (config) {
+  // See https://www.11ty.dev/docs/config/
+  config.setInputDirectory('content');
+  config.setOutputDirectory('../public');
```

Comment:
The `public` folder is currently created in the project root. IMO it would be better to have it in `api-docs`.

The config is confusing me as those paths seem to point at the project root as well, but the folders are actually in `api-docs`.

Reply:
Done.